### PR TITLE
Add release automation

### PR DESCRIPTION
**Pull request by timja**
Followed: https://code.visualstudio.com/api/working-with-extensions/continuous-integration#automated-publishing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
     "test-watch": "npm run test-compile -- -watch",
     "test-compile": "tsc -p ./",
     "pretest": "npm run test-compile",
-    "test": "node ./out/test/runTest.js"
+    "test": "node ./out/test/runTest.js",
+    "deploy": "vsce publish --yarn"
```
Comment (sladyn98) [inline package.json:56]: Would we put in a place holder for the version number

Comment (timja) [reply package.json:56]: version number is based on the package.json

Comment (sladyn98) [reply package.json:56]: Sounds cool :+1:

**Review by sladyn98 — APPROVED** [review]

File: azure-pipelines.yml [hunk 2]
```diff
@@ -32,3 +35,7 @@ steps:
     displayName: Run Tests
     env:
       DISPLAY: ':99.0'
+  - bash: |
+      yarn deploy -p $(VSCODE_MARKETPLACE_TOKEN)
```
Comment (jetersen) [inline azure-pipelines.yml:39]: Somewhat concerned if the secret is exposed like that, but if the doc says it is. Let it be so.

**Review by jetersen — APPROVED** [review]
LGTM

**Review by jetersen — CHANGES_REQUESTED** [review]
If you use yarn to build and publish, we should have an yarn.lock actually.
Remember to remove package-lock.json
And udate scripts section to use yarn instead of npm

File: package.json [hunk 3]
```diff
@@ -52,7 +52,8 @@
     "test-watch": "npm run test-compile -- -watch",
     "test-compile": "tsc -p ./",
     "pretest": "npm run test-compile",
```
Comment (jetersen) [inline package.json]: The scripts should be updated to use yarn as well imho

File: .vscodeignore [hunk 4]
```diff
@@ -17,3 +17,5 @@ images/**
 .editorconfig
 webpack.config.js
 node_modules/**
+yarn.lock
```
Comment (jetersen) [inline .vscodeignore:20]: added some more ignore files.

**Review by jetersen — APPROVED** [review]
LGTM

**Comment by jetersen** [comment]
bumping version can be done via `yarn version` and you could add some `postversion` script that runs tests, and pushes tags.

**Pull request merged**